src/tex/ms.tex

```diff
@@ -138,7 +138,7 @@ \section{Prior}
         $d_h$/\si{\angstrom} & $10.0$ & $[8.0, 16.0)$\\
         $V_h$/\si{\angstrom^3} & $339.5$ & $[300.0, 380.0)$\\
         $d_t$/\si{\angstrom} & $21.0$ & $[10.0, 26.0)$\\
-        $\phi_t$ & $1.0$ & $[0.5, 1.0)$\\
+        $\phi_t$ & $1.0$ & $[0.5, 1.0]$\\
```

Comment:
Why this change? The [ ) syntax is used to show that the start point of the interval is included while the end point is not.

Reply:
I see. My point here is that in principle you shouldn't get value of 1.0 if it is `)` but I guess it is due to rounding? Maybe we can set it to e.g. 0.9 to avoid any confusion.

Reply:
`[0.5, 1.0)` indicates that the minimum is 0.5 and the maximum is 0.99999999... so that is accounting for the fact that 1.0 is impossible. While 0.9 removes any confusion the table reproduced from a table in another paper so it should match.